src/url-utils.ts

```diff
@@ -132,6 +159,9 @@ export function rewriteHrefBaseUrl<T>(
   return relativeUrl as FileRelativeUrl;
 }
 
+/**
+ * Ensures a leading slash on given string.
```

Comment:
Would probably call this `ensureLeadingSlash`, since otherwise you might thing it was like `path.resolve`

Reply:
Fixed d617c6319723daf66df233c15197417abc518131